### PR TITLE
313

### LINKED ISSUE #311: Redirect on child pages of /jp/sustainability/environment/*
https://main--sunstar--hlxsites.hlx.live/jp/sustainability/environment

If you try to click on any of the following links:
image

It goes to this:

image

### PR DESCRIPTION
**Pull request by ehrro**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #311 

Test URLs:
- Original: https://www.sunstar.com/sustainability/social/employees
- Before: https://main--sunstar--hlxsites.hlx.live/sustainability/social/employees
- After: https://313--sunstar--hlxsites.hlx.live/sustainability/social/employees

- [ ] New Blocks introduced in this PR
      If yes, please provide details below

Text   | Documentation
------------- | -------------
 For e.g. cards | [doc-link](https://main--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/cards&index=0)

- [X] New variations introduced in this PR

text(emphasis)    | Documentation
------------- | -------------
 For e.g. cards (grid)  | [doc-link](https://main--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/cards&index=2)

image

### CODE REVIEW

File: blocks/text/text.css [hunk 1]
```diff
@@ -73,9 +73,39 @@ main .block.text.lead {
...
+  margin-bottom: 0;
+}
+
 @media (min-width: 77rem) {
   main .block.text.lead {
     font-size: var(--body-font-size-xxl);
   }
+
+  main .block.text.emphasis {
+    background-color: var(--light);
```
Comment (mkbansal1) [inline blocks/text/text.css:97]: background is already set in line no 79, it can be avoided to override.

File: blocks/text/text.css [hunk 2]
```diff
@@ -73,9 +73,39 @@ main .block.text.lead {
...
+    margin-inline: -3rem;
+    padding: 3rem;
+    margin-block: 2rem;
+  }
+
+  main .block.text.emphasis > div > div > :first-child {
+    margin-top: 0;
```
Comment (mkbansal1) [inline blocks/text/text.css:104]: this rule can be avoided as same is defined in line 83

File: blocks/text/text.css [hunk 3]
```diff
@@ -73,9 +73,39 @@ main .block.text.lead {
...
+  }
+
+  main .block.text.emphasis > div > div > :last-child {
+    margin-bottom: 0;
```
Comment (mkbansal1) [inline blocks/text/text.css:108]: this rule can be avoided as same is defined in line 87

**Review by mkbansal1 — CHANGES_REQUESTED** [review]
@ehrro I have advised some optimization. Please check my comments.

**Comment by sdmcraft** [comment]
The new variant info provided in the description does not seem to have corresponding entries in block library.

**Comment by ehrro** [comment]
Hello @sdmcraft I created both entries in sidekick library. Text(lead) and Text(emphasis)

image

**Review by sdmcraft — APPROVED** [review]

**Review by ehrro — COMMENTED** [review]
I guess it's all good I don't know how to use this :D

**Pull request merged**